### PR TITLE
Change the name for npm

### PR DESCRIPTION
**Pull request by marimeireles**
According to Sylvain the npm package shouldn't have ipy as a prefix, so I'm changing.
@martinRenou could you please check to see if this is all I need to change on the js side?
Thanks! 🌸

### CODE REVIEW

File: jupyter-cytoscape.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (martinRenou) [inline jupyter-cytoscape.json]: I guess you would need to remove the old `ipycytoscape.json` file?

Comment (marimeireles) [reply jupyter-cytoscape.json]: right!

File: jupyter-cytoscape.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  "load_extensions": {
+    "ipycytoscape/extension": true
```
Comment (maartenbreddels) [inline jupyter-cytoscape.json]:
```suggestion
    "jupyter-cytoscape/extension": true
```
I think this should be the extension name right?

Comment (marimeireles) [reply jupyter-cytoscape.json]: Yes

**Pull request merged**